e2e/cypress/integration/messaging/collapse_link_spec.js

```diff
@@ -0,0 +1,93 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [number] indicates a test step (e.g. 1. Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+describe('Messaging', () => {
+    before(() => {
+        // # Set the configuration on Link Previews
+        cy.apiUpdateConfig({
+            ServiceSettings: {
+                EnableLinkPreviews: true,
+            },
+        });
+
+        cy.apiLogin('user-1');
+        cy.apiSaveLinkPreviewPreference(JSON.stringify(true));
+
+        cy.apiLogin('sysadmin');
+        cy.apiSaveLinkPreviewPreference(JSON.stringify(true));
+
+        // # Login and go to /
+        cy.apiLogin('user-1');
+        cy.visit('/');
+    });
+
+    it('M18708-Link preview - Removing it from my view removes it from other user\'s view', () => {
+        const message = 'https://www.bbc.com/news/uk-wales-45142614';
+
+        // # Create new DM channel with user's email
+        cy.visit('/ad-1/channels/town-square');
+
+        // # Post message to use
+        cy.postMessage(message);
+
+        cy.getLastPostId().then((postId) => {
+            // * Check the preview is shown
+            cy.get(`#${postId}_message`).find('.file-preview__button').should('exist');
```

Comment:
Let's use `.attachment--opengraph` to check link preview and `.attachment__image` to check image preview on link.  ``.file-preview__button`` sounds different and might change in the future.

Since the test actually verifies **link** and **image** previews.  Please add initial assertion for those.

```suggestion
            // * Link and image previews are shown for sysadmin
            cy.get(`#${postId}_message`).find('.attachment--opengraph').should('exist');
            cy.get(`#${postId}_message`).find('.attachment__image').should('exist');
```